### DISCUSSION TITLE
Does `configure med fast-start tx-interval 0` make sense?

### DISCUSSION THREAD

**Discussion by systemcrash** [lldpd/lldpd#636] [Q&A]
Or must the value be larger than zero?

The code doesn't seem to indicate that it's invalid. 

https://github.com/lldpd/lldpd/blob/55a65f5e4111503b365a19a7377794ec717e4b10/src/client/conf-med.c#L458

Or, in other words: a zero second fast-start sends 4 packets immediately?

**Comment by vincentbernat** [comment]
Yes, it should work.

**Comment by systemcrash** [comment]
👍
